cs/nuget/scripts/10-install-stack.ps1

```diff
@@ -0,0 +1,24 @@
+[CmdletBinding()]
+param ( )
+
```

Comment:
Consider moving this script to `tools/windows/release`, as we're now putting these sort of scripts under `tools`.

Reply:
Good point